pkg/cmd/repo/view/view.go

```diff
@@ -114,6 +115,8 @@ func viewRun(opts *ViewOptions) error {
 		return err
 	}
 
+	opts.IO.ResolveBgColor()
```

Comment:
Nit: perhaps DetectTerminalBackground would be better:
- "detect" implies an activity, an in this case it's sending terminal escape sequences
- "bg" reads weird when other words are spelled in full
- "color" implies that we are reading out the terminal color precisely, whereas we're only detecting whether it's dark or light

Reply:
I am going to go with `DetectTerminalTheme` 👍